org.zrythm.Zrythm.json

```diff
@@ -0,0 +1,442 @@
+{
+    "app-id": "org.zrythm.Zrythm",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "zrythm_launch",
+    "rename-icon": "zrythm",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        
+        "--share=network",
+        
+        "--socket=pulseaudio",
+        "--device=all",
+        
+        "--filesystem=xdg-run/pipewire-0",
+        "--system-talk-name=org.freedesktop.RealtimeKit1",
+        
+        "--env=LXVST_PATH=/app/extensions/Plugins/lxvst",
+        "--env=VST3_PATH=/app/extensions/Plugins/vst3",
+        "--env=LV2_PATH=$HOME/.lv2:/app/lib/lv2:/app/extensions/Plugins/lv2"
```

Comment:
This actually doesn't work as $HOME is never expanded. So we end up having to set it in a wrapper script.

Reply:
I only included `$HOME/.lv2` due to instructions [here](https://github.com/flathub/org.freedesktop.LinuxAudio.BaseExtension/tree/branch/21.08#application-using-audio-plugins). But it does seem to already succesfully pick up plugins from the correct `lv2` directory, so I don't know if it's strictly necessary.

Technically there's a wrapper script already upstream (`zrythm_launch` vs the actual `zrythm` binary). So I guess I could `sed` it, or it could be adjusted somehow upstream.

Reply:
that doc is wrong. I know I wrote it. It assumed that variable expansion was done, while it's not.

Reply:
some observations/questions:

1. `LXVST_PATH` is not recognized by Zrythm. `VST_PATH` for VST2 and `VST3_PATH` for VST3
2. what happens if the user passes e.g. `LV2_PATH`? are these appended or do they override the user's environment?
3. I can hardcode these paths along with the rest so no messing with environment is needed. can flatpak plugins work with non-flatpak hosts too? or do these only work in flatpak hosts?

Reply:
2. that's a question for upstream to answer.

Reply:
I meant if you run `LV2_PATH=/mypath flatpak run`, what do these `--env` instructions do? do the re-set environment variables given by the user?

Reply:
I hardcoded those paths when `for_flathub` option is passed so the user can still override them with env variables

Reply:
Seems to work for vst, vst3, and lv2 plugins now, so resolving this.